spec/controllers/v0/dependents_applications_controller_spec.rb

```diff
@@ -52,7 +53,23 @@
         VCR.use_cassette('bgs/dependent_service/submit_686c_form') do
           post(:create, params: test_form)
         end
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'with valid params v2' do
+      before do
+        Flipper.enable(:va_dependents_v2)
```

Comment:
You'll want to do the `allow(Flipper).to receive(:enabled?)...` like you do elsewhere in this PR.